core/src/main/resources/hudson/model/ListView/listView.js

```diff
@@ -0,0 +1,8 @@
+Behaviour.specify("#recurse", 'ListView', 0, function (e) {
+    var nestedElements = $$('SPAN.nested')
+    e.onclick = function () {
+        nestedElements.each(function (el) {
+            e.checked ? el.show() : el.hide();
+        });
+    }
+});
```

Comment:
The red icon means that you have no empty line at the end of the file. This will complexify a bit the merge from other authors. In general it's not that important, just a matter of good practice.

Reply:
empty line added